Sources/ArgumentParser/Parsable Properties/OptionGroup.swift

```diff
@@ -84,7 +85,17 @@ extension OptionGroup: CustomStringConvertible {
     case .value(let v):
       return String(describing: v)
     case .definition:
+      if let hidden = _hidden {
+        return "OptionGroup(*definition*) _hidden: \(hidden)"
+      }
```

Comment:
This works, but we don't really want to pass configuration values through this type's string representation. Instead, let's add `var _hiddenFromHelp: Bool { get }` as a protocol requirement in `ArgumentSetProvider`, here: https://github.com/apple/swift-argument-parser/blob/main/Sources/ArgumentParser/Parsable%20Types/ParsableArguments.swift#L230

You can add a default implementation that always returns `false`, and then this type will have the customized version, defined above.